tests/nlu/data_augmentation/test_augmenter.py

```diff
@@ -0,0 +1,332 @@
+import os
+from typing import Callable
+
+from _pytest.pytester import RunResult, Testdir
+from rasa.nlu.components import Component
+from rasa.nlu.constants import TOKENS_NAMES
+from rasa.shared.constants import LATEST_TRAINING_DATA_FORMAT_VERSION
+from rasa.shared.nlu.training_data.message import Message
+import rasa.shared.nlu.training_data.loading
+import rasa.shared.utils.io
+from rasa.shared.nlu.constants import (
+    INTENT,
+    TEXT,
+    VOCABULARY,
+)
+
+
+def test_augmenter_create_tokenizer(run: Callable[..., RunResult],):
+    from rasa.nlu.data_augmentation.augmenter import _create_tokenizer_from_config
+
+    data_root = os.path.dirname(
+        os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+    )
+    config_path = os.path.join(data_root, "data/test_nlu_paraphrasing/config.yml")
+
+    tokenizer = _create_tokenizer_from_config(config_path=config_path)
+
+    assert isinstance(tokenizer, Component)
+
+    # Test Config has a simple WhitespaceTokenizer
+    expected_tokens = ["xxx", "yyy", "zzz"]
+    message = Message(data={TEXT: "xxx yyy zzz", INTENT: "abc"})
+
+    tokenizer.process(message)
+    tokens = [token.text for token in message.get(TOKENS_NAMES[TEXT])]
+
+    assert tokens == expected_tokens
+
+
+def test_augmenter_create_tokenizer_empty_config(run: Callable[..., RunResult],):
+    from rasa.nlu.data_augmentation.augmenter import _create_tokenizer_from_config
+
+    data_root = os.path.dirname(
+        os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+    )
+    config_path = os.path.join(data_root, "data/test_nlu_paraphrasing/empty_config.yml")
+
+    # Should fallback to a WhitespaceTokenizer
+    tokenizer = _create_tokenizer_from_config(config_path=config_path)
+
+    assert isinstance(tokenizer, Component)
+
+    # Test Config has a simple WhitespaceTokenizer
+    expected_tokens = ["xxx", "yyy", "zzz"]
+    message = Message(data={TEXT: "xxx yyy zzz", INTENT: "abc"})
+
+    tokenizer.process(message)
+    tokens = [token.text for token in message.get(TOKENS_NAMES[TEXT])]
+
+    assert tokens == expected_tokens
```

Comment:
As i mentioned before, this should trigger an exception, so we should test for that.